mainframer.sh

```diff
@@ -36,83 +36,31 @@ if [ -z "$BUILD_COMMAND" ]; then
 	exit 1
 fi
 
-# Create build folder in case if it does not exist.
-mkdir -p "$PROJECT_DIR"/build
-
-# Remove previous archives of the project.
-rm -f "$PROJECT_DIR"/build/project_for_remote_build.tar "$PROJECT_DIR"/build/remotely_built_project.tar
-
-# Archiver.
-
-DETECT_ARCHIVER="if type 'pigz' > /dev/null; then echo 'pigz'; else echo 'gzip'; fi"
-LOCAL_ARCHIVER=`eval $DETECT_ARCHIVER`
-
-# Archive project.
 pushd "$PROJECT_DIR"
-LOCAL_ARCHIVE_COMMAND="tar \
--c \
---exclude='build/project_for_remote_build.tar' \
---exclude='local.properties' \
```

Comment:
You must exclude `local.properties`, otherwise remote build may fail if it'll see your `local.properties` with path to android sdk that does not make sense for remote build machine!